tests/Cargo.toml

```diff
@@ -23,7 +23,7 @@ mixnet-topology = { path = "../mixnet/topology" }
 # Using older versions, since `mixnet-*` crates depend on `rand` v0.7.3.
 rand = "0.7.3"
 once_cell = "1"
-secp256k1 = { version = "0.26", features = ["rand"] }
+secp256k1 = { version = "0.28", features = ["rand"] }
```

Comment:
Why do we update this here? Are we using something not available in other versions? Otherwise it would be better if it has its own pr.

Reply:
This should be updated by vscode plugin. I will change it back.